tests/test-war-smoke/src/test/java/com/google/cloud/runtimes/jetty/test/smoke/AnnotationIntegrationTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.runtimes.jetty.test.smoke;
+
+import com.google.cloud.runtime.jetty.test.AbstractIntegrationTest;
+import com.google.cloud.runtime.jetty.test.annotation.LocalOnly;
+import com.google.cloud.runtime.jetty.test.annotation.RemoteOnly;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class AnnotationIntegrationTest extends AbstractIntegrationTest {
```

Comment:
sure, but then we are running some sort of container in there for local and remote tests, etc